src/query/list_file.c

```diff
@@ -5229,8 +5176,13 @@ qfile_allocate_list_cache_entry (int req_size)
 
       (void) memset ((void *) &pent->s.entry, 0, sizeof (QFILE_LIST_CACHE_ENTRY));
     }
+  else
+    {
+      return NULL;
```

Comment:
The following pthread_mutex_init will not be called

Reply:
this code will be fixed not to the mutex.